shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -140,38 +140,431 @@ mod udt {
+            ColType::Set => "set",
+            ColType::Udt => "udt",
+            ColType::Tuple => "tuple",
+            ColType::Null => "null",
+        }
+    }
+
+    fn get_type_example(col_type: ColType) -> &'static str {
+        match col_type {
+            ColType::Ascii => "'ascii string'",
+            ColType::Bigint => "1844674407370",
+            ColType::Blob => "bigIntAsBlob(10)",
+            ColType::Boolean => "true",
+            ColType::Counter => "12",
+            ColType::Decimal => "1.111",
+            ColType::Double => "1.11",
+            ColType::Float => "1.11",
+            ColType::Int => "1",
+            ColType::Timestamp => "'2011-02-03 04:05+0000'",
+            ColType::Uuid => "84196262-53de-11ec-bf63-0242ac130002",
+            ColType::Varchar => "'varchar'",
+            ColType::Varint => "198121",
+            ColType::Timeuuid => "84196262-53de-11ec-bf63-0242ac130002",
+            ColType::Inet => "'127.0.0.1'",
+            ColType::Date => "'2011-02-03'",
+            ColType::Time => "'08:12:54'",
+            ColType::Smallint => "32767",
+            ColType::Tinyint => "127",
+            _ => panic!("dont have an example for {}", col_type),
+        }
+    }
+
+    fn get_type_example_result_value(col_type: ColType) -> ResultValue {
+        match col_type {
+            ColType::Ascii => ResultValue::Ascii("ascii string".into()),
+            ColType::Bigint => ResultValue::BigInt(1844674407370),
+            ColType::Blob => ResultValue::Blob(vec![0, 0, 0, 0, 0, 0, 0, 10]),
+            ColType::Boolean => ResultValue::Boolean(true),
+            ColType::Counter => ResultValue::Counter(12),
+            ColType::Decimal => ResultValue::Decimal(vec![0, 0, 0, 3, 4, 87]),
+            ColType::Double => ResultValue::Double(vec![63, 241, 194, 143, 92, 40, 245, 195]),
+            ColType::Float => ResultValue::Float(vec![63, 142, 20, 123]),
+            ColType::Int => ResultValue::Int(1),
+            ColType::Timestamp => ResultValue::Timestamp(vec![0, 0, 1, 45, 233, 177, 205, 224]),
+            ColType::Uuid => ResultValue::Uuid(
+                uuid::Uuid::parse_str("84196262-53de-11ec-bf63-0242ac130002").unwrap(),
+            ),
+
+            ColType::Varchar => ResultValue::Varchar("varchar".into()),
+            ColType::Varint => ResultValue::VarInt(vec![3, 5, 233]),
+            ColType::Timeuuid => ResultValue::TimeUuid(
+                uuid::Uuid::parse_str("84196262-53de-11ec-bf63-0242ac130002").unwrap(),
+            ),
+            ColType::Inet => ResultValue::Inet("127.0.0.1".into()),
+            ColType::Date => ResultValue::Date(vec![128, 0, 58, 160]),
+            ColType::Time => ResultValue::Time(vec![0, 0, 26, 229, 187, 195, 188, 0]),
+            ColType::Smallint => ResultValue::SmallInt(32767),
+            ColType::Tinyint => ResultValue::TinyInt(127),
+            _ => panic!("dont have an example for {}", col_type),
+        }
+    }
+
+    mod list {
```

Comment:
this list module seems weird, not everything in here is just for lists...

Reply:
It's inserting a list of other collections, so technically it is testing lists. I have other work waiting to go in to test maps and sets in the same way.

Reply:
nah, if theres context im missing from future PRs thats fine, leave it as is.
We can always refactor it later if we decide it looks weird after everything is in.